go-selinux/selinux_linux_test.go

```diff
@@ -165,15 +172,13 @@ func TestSELinux(t *testing.T) {
 	if err == nil {
 		t.Log(FSCreateLabel())
 	} else {
-		t.Log("SetFSCreateLabel failed", err)
-		t.Fatal(err)
+		t.Fatal("SetFSCreateLabel failed:", err)
```

Comment:
Nit; `t.Fatal` terminates the test, so we could get rid of the if/else, and instead put the `t.Fatal` in a `if err != nil` branch

Reply:
I was just trying to minimize the changes to simplify review. But since you've asked for it... 😁 

Indeed it looks better this way.